twitter/table_twitter_user.go

```diff
@@ -49,19 +50,34 @@ func listUser(ctx context.Context, d *plugin.QueryData, _ *plugin.HydrateData) (
 		plugin.Logger(ctx).Error("twitter_user.listUser", "query_error", lookupErr, "opts", opts)
 		return nil, lookupErr
 	}
+
+	softError := []*twitter.ErrorObj{}
+
 	// Soft error, e.g. 404
 	if len(result.Raw.Errors) > 0 {
-		errMsgs := []string{}
 		for _, e := range result.Raw.Errors {
 			plugin.Logger(ctx).Error("twitter_user.listUser", "query_error", e, "opts", opts)
-			errMsgs = append(errMsgs, fmt.Sprintf("%s: %s", e.Title, e.Detail))
+			
+			// Check if the Not Found Error is a result of "pinned_tweet_id" parameter, return nil
```

Comment:
Why do we need to handle a not found error from the `pinned_tweet_id` parameter differently?

Reply:
@cbruno10 The issue occurs for the pinned_tweet column since the pinned_tweet_id (935639732112982017) doesn't exist. That's the reason why we need to handle the not found error differently.